{"uuid":"d0b5e8b1-0f3d-417b-81a1-74c833d3b8ff","name":"Scenario Outline: To validate if user getting error message when enter valid python code in operations stack page","children":["c4bc6696-58a3-4d52-87da-6a8c138b59ec"],"befores":[],"afters":[{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065954332,"stop":1709065954333},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065954954,"stop":1709065954955},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065956335,"stop":1709065956335},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065956700,"stop":1709065956701},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065956954,"stop":1709065956955},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065957603,"stop":1709065957604},{"name":"stepDefinitions.ApplicationHooks.addScreenshot(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065957841,"stop":1709065957842},{"name":"stepDefinitions.ApplicationHooks.tearDown()","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1709065957842,"stop":1709065957955}],"start":1709065947686,"stop":1709065957955}